src/app/not-found.js

```diff
@@ -1,7 +1,16 @@
+import Image from "next/image";
+import fish from "@/public/fish.webp";
+import Button from "@/components/Button";
+
 const NotFound = () => {
   return (
-    <div>
-      <p>404 Page Not Found</p>
+    <div className="flex flex-col justify-center items-center">
+      <p className="text-4xl font-bold"> - Oops!</p>
```

Comment:
```suggestion
      <p className="text-4xl font-bold">Oops!</p>
```

maybe we can leave out the dash?